pom.xml

```
@@ -42,7 +42,7 @@
         <changelist>-SNAPSHOT</changelist>
         <jenkins.version>2.222.4</jenkins.version>
         <java.level>8</java.level>
-        <subversion-plugin.version>2.14.0</subversion-plugin.version>
+        <subversion-plugin.version>2.14.3-rc1161.5aa3d92d94e9</subversion-plugin.version> <!-- TODO: https://github.com/jenkinsci/subversion-plugin/pull/257 -->
```

The PR was merged and released so

```suggestion
        <subversion-plugin.version>2.14.3</subversion-plugin.version>
```

---

Going to bump subversion to 2.14.4 to take in https://github.com/jenkinsci/subversion-plugin/pull/259 where digester is removed completely. See https://github.com/jenkinsci/subversion-plugin/pull/258 for context

---

Closing in favor of https://github.com/jenkinsci/workflow-scm-step-plugin/pull/52